forge/src/executor/inspector/mod.rs

```diff
@@ -0,0 +1,19 @@
+#[macro_use]
+mod macros;
+
+mod logs;
+pub use logs::LogCollector;
+
+use ethers::abi::RawLog;
+
+// TODO: Move
+#[derive(Debug)]
+pub struct ExecutorState {
+    pub logs: Vec<RawLog>,
+}
```

Comment:
This will contain state that we want to preserve after a call (e.g. logs, traces). The state will be shared between all inspectors. I'm not sure where to place it so it lives here for now